Infrastructure/kind.yaml

```diff
@@ -12,4 +12,7 @@ nodes:
       - containerPort: 30001
         hostPort: 9000
         # Limit the traefik dashboard to localhost
-        listenAddress: "127.0.0.1"
\ No newline at end of file
+        listenAddress: "127.0.0.1"
+    extraMounts:
+      - hostPath: mnt/data
+        containerPath: /data
```

Comment:
This binds the `/data` folder in the Kubernetes node to a folder on the host machine.  Note that since this means changing the database passwords in `values.yaml` or `helmfile.yaml` won't work without first deleting old database from the host machine.